### PR TITLE
doc: clarify BOARD_ROOT documentation

### PR DESCRIPTION
**Pull request by hallo-alex**
clarify BOARD_ROOT must point to dir ABOVE boards

### CODE REVIEW

File: doc/application/index.rst [hunk 1]
```diff
@@ -580,7 +580,7 @@ targeting this board by specifying the location of your custom board
 information with the ``-DBOARD_ROOT`` parameter to the CMake
 build system::
 
-   cmake -DBOARD=<board name> -DBOARD_ROOT=<path to boards> ..
+   cmake -DBOARD=<board name> -DBOARD_ROOT=<path the directory containig a "boards" directory> ..
```
Comment (ulfalizer) [inline doc/application/index.rst]: Language nit: "path to directory containing a boards/ directory"

Comment (dbkinder) [reply doc/application/index.rst]: ... also fixes the misspelling of "containing"  :)

**Review by dbkinder — CHANGES_REQUESTED** [review]

**Review by dbkinder — CHANGES_REQUESTED** [review]
There are lots of index.rst files in the documentation. The PR title should be more informative, something like:
```
doc: clarify BOARD_ROOT documentation
```

**Review by dbkinder — APPROVED** [review]

**Comment by aescolar** [comment]
Please squash both commits together. In general, when fixing a pull request commit, instead of creating a new one, amend the previous one.
Also please address the issue reported by gitlint (you need to add a signed off message).

**Comment by aescolar** [comment]
@hallo-alex : Please note that until the issues reported in https://github.com/zephyrproject-rtos/zephyr/pull/14021#issuecomment-469057614 are not fixed, and the commits are not squashed this commit PR cannot be accepted.

**Review by aescolar — CHANGES_REQUESTED** [review]
Please:
* squash both commits together
* Add a signed off line to the commit message (you can use "git commit --amend -s")

**Pull request closed without merging**